### PR TITLE
User page

### LINKED ISSUE #83: Clean up color scheme for light and dark mode, with material-ui
On dark mode, text is too dark

### LINKED ISSUE #99: Fix breaking change on ShowForm userID fetch
Please don't commit any changes without testing them, but especially don't make changes to components that you've never worked on AND don't test them.

[PR#87](https://github.com/BlueOcean37/Bravo/pull/87)
[commit](https://github.com/BlueOcean37/Bravo/pull/87/commits/ce01a3b889f1dee44b0fe0874a70d9d680f0afaf)

### PR DESCRIPTION
**Pull request by diegochamilton**
This commit includes

1. dark mode fully functional and tested
2. styling issues fixed such as same width on user and show page.
3. scroll on home page for reviews
4. only members can write review

### CODE REVIEW

**Review by WarrenWongCodes — APPROVED** [review]
Looks good

File: client/src/components/authentication/signup/signup.module.scss [hunk 1]
```diff
@@ -1,14 +1,19 @@
-#formContainer{
-  display:flex;
+#formContainer {
+  display: flex;
   flex-direction: column;
   min-width: 300px;
   max-width: 900px;
+  overflow: scroll;
```
Comment (timapar) [inline client/src/components/authentication/signup/signup.module.scss]: Adding the overflow rule creates hideous scroll bars on the component even when not needed. I understand that you are trying to make it more accessible to different screen sizes/resolutions, but I think there has to be a better way. Also, there isn't a screen out there that has web browsing capabilities that has a resolution less than 300px, so this size shouldn't really be a problem. But, we can also look into changing to em/rem units if you want.

File: client/src/components/homeView/home.module.scss [hunk 2]
```diff
@@ -17,5 +17,7 @@
 }
 
 #reviewsContainer {
+  max-height: 150vh;
+  overflow: scroll;
```
Comment (timapar) [inline client/src/components/homeView/home.module.scss]: Again, the added scroll bars really diminish the aesthetics of the site and Johnny explicitly asked for the scroll bars on the reviews to go away.

File: client/src/components/homeView/homeshows.module.scss [hunk 3]
```diff
@@ -6,47 +6,14 @@
 .imgContainer {
   display: flex;
   justify-content: space-between;
-  width: 250px;
-  height: 400px;
+  min-width: 100px;
```
Comment (timapar) [inline client/src/components/homeView/homeshows.module.scss]: I'm hesitant to use min-s here. Without fixed sizes, I worry images that are different aspect ratios and different resolutions will be displayed differently. I think we want all of the cards to be uniform and the images themselves adjusted to fit inside. Perhaps switching width and height to vw and vh can achieve a similar outcome?

File: client/src/components/header/navbar/NavBar.jsx [hunk 4]
```diff
@@ -83,63 +83,69 @@ export default function NavBar() {
...
+// export const light = {
+//   palette: {
+//     type: 'light',
+//   },
+// };
+// export const dark = {
+//   palette: {
+//     type: 'dark',
+//   },
+// };
```
Comment (timapar) [inline client/src/components/header/navbar/NavBar.jsx]: Lots of comments left in this file

File: client/src/components/showForm/ShowForm.jsx [hunk 5]
```diff
@@ -149,7 +150,6 @@ export default function ShowForm() {
             value={date}
             required
             type="date"
-            onChange={(e) => setDate(e.target.value)}
```
Comment (timapar) [inline client/src/components/showForm/ShowForm.jsx:152]: Why did you delete this? This needs to be undone

File: client/src/components/showForm/showForm.module.scss [hunk 6]
```diff
@@ -25,3 +25,11 @@
   align-items: center;
   margin: 20px;
 }
+
```
Comment (timapar) [inline client/src/components/showForm/showForm.module.scss]: Seems like you were testing something. Do these rule sets need to stay in here? They aren't being applied anywhere as far as I can tell.

**Review by timapar — CHANGES_REQUESTED** [review]

File: client/src/components/homeView/homeshows.module.scss (hunk 3 above)
Comment (diegochamilton) [reply client/src/components/homeView/homeshows.module.scss]: done. fixed it with vw and vh

File: client/src/components/header/navbar/NavBar.jsx (hunk 4 above)
Comment (diegochamilton) [reply client/src/components/header/navbar/NavBar.jsx]: deleted

File: client/src/components/showForm/ShowForm.jsx (hunk 5 above)
Comment (diegochamilton) [reply client/src/components/showForm/ShowForm.jsx:152]: i put it back in

File: client/src/components/showForm/showForm.module.scss (hunk 6 above)
Comment (diegochamilton) [reply client/src/components/showForm/showForm.module.scss]: deleted

**Review by timapar — APPROVED** [review]
Thank you for making those changes. I think the home page show cards look great!
